src/06-loops.js

```diff
@@ -61,7 +69,11 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+function containsWhileLoop(array, target) {
+  while (i = array[target]){
```

Comment:
Be careful with our comparison here.  We are DECLARING i to equal the value of array[target] for every iteration.  I do this a lot :)